question-service/controller/question-controller.js

```diff
@@ -0,0 +1,55 @@
+import QuestionModelSchema from "../model/question-model.js";
+
+export async function getQuestion(request, response) {
```

Comment:
is this function still needed?

Reply:
Commented it out, might require for future use.